blazeutils/spreadsheets.py

```diff
@@ -252,15 +291,36 @@ def nextrow(self):
         self.colnum = 0
 
 
+class WriterX:
+    if openpyxl:
+        writer_cls = _OpenpyxlWriter
+    else:
+        writer_cls = _XlsxwriterWriter
```

Comment:
It seems reasonable to let the developer choose the library. If a 3rd-party depedency brought in openpyxl, for example, that would break any existing code depending on the xlsxwriter implementation.